Add support for Azure AI Search (Azure Cognitive Search) (#14925)
OrchardCMS/OrchardCore: src/docs/reference/modules/AzureCognitiveService/README.md

# Azure Cognitive Search (`OrchardCore.Search.Azure.CognitiveService`)

The Azure Cognitive Search module allows you to manage Azure Cognitive Search indices.

Before enabling the service, you'll need to configure the connection to the server. You can do that by adding the following into `appsettings.json` file

```
{
  "OrchardCore":{
    "OrchardCore_CognitiveSearch_Azure":{
      "Endpoint":"https://[search service name].search.windows.net",
      "IndexesPrefix":"",
      "Credential":{
        "Key":"the server key goes here"
      }
    }
  }
}
```

Then navigate to `Search` > `Indexing` > `Azure Cognitive Indices` to add an index.

## Search Module (`OrchardCore.Search`)

> When the Search module is enabled along with Azure Cognitive Search, you'll be able to use run the frontend site search against your Azure Cognitive Search indices.

Reviewer: ```suggestion
When the `Search` module is enabled along with `Azure Cognitive Search`, you can run the frontend site search against your Azure Cognitive Search indices.
```
